src/python/gudhi/simplex_tree.pyx

```diff
@@ -812,16 +812,19 @@ cdef class SimplexTree:
 
     def __eq__(self, other:SimplexTree):
         """Test for structural equality
+
         :returns: True if the 2 simplex trees are equal, False otherwise.
```

Comment:
A bit redundant. We can keep it (or try to rephrase "if the 2 complexes have the same simplices with the same filtration values", although ISTR it isn't exactly that?), but removing it seems clear enough.

Reply:
Remove redundancy on [a2ac2e5](https://github.com/GUDHI/gudhi-devel/pull/871/commits/a2ac2e5867e9fca8a93fafe733f96c8f5d7cf714)